directives/annotated.py

```diff
@@ -357,6 +365,22 @@ def annotate(html, section_name, annotations):
     content = u''.join(result)
     return u''.join([parts[0], parts[1], content, parts[3], parts[4]])
 
+def add_assets(app):
+    app.add_stylesheet(CSS_FILE)
+    app.add_javascript(JS_FILE)
+
+def copy_assets(app, exception):
+    if app.builder.name not in ['html', 'readthedocs'] or exception:
+        return
+    logger = logging.getLogger(__name__)
+    logger.info('Copying aplus annotated stylesheet/javascript... ', nonl=True)
+    dest = os.path.join(app.builder.outdir, '_static', CSS_FILE)
```

Comment:
The directory name `_static` should not be hardcoded since it can be modified in the `conf.py` file. You should be able to read the value from `app.config` or something like that.

Reply:
I think this _static folder is always created by Sphinx, and therefore, it should be harcoded.
https://github.com/sphinx-doc/sphinx/issues/2202

Reply:
Right, seems like you are correct. So the build output directory may use a hardcoded `_static` directory while the static "source" directory under the course repo may be renamed in `conf.py`.